dockerfiles/theia/Dockerfile

```diff
@@ -11,7 +11,7 @@
 FROM centos
 
 ARG GITHUB_TOKEN
-ARG THEIA_VERSION=0.3.14
+ARG THEIA_VERSION=plugin-id
```

Comment:
what is `plugin-id` ?

Reply:
@riuvshin the name of the branch (like 0.3.14)